src/erc1155.js

```diff
@@ -0,0 +1,148 @@
+import getWeb3 from "./web3/getWeb3";
+import log from 'loglevel';
+import axios from 'axios';
+
+export default class ERC1155 {
+
+  getBalance() {
+    let headersList = {
+      "Accept": "*/*"
+    }
+    let reqOptions = {
+      url: "https://api.dev.theapis.io/api/v1/tokens/0x6A3Cd6a4f7288a43921Ff88796cb75D9e36A8b9A/erc1155/balance?tokenAddress=0x85c95902239797b69beab4e617411fb94981a2a9&type=erc1155&id=1&apiKey=58fc282a-498f-4695-88ac-f694fa98df9d&chain=ethereum&network=rinkeby",
+      method: "GET",
+      headers: headersList,
+    }
```

Comment:
These `reqOptions` may be passed directly to the function as an anonymous object.